channel-fuel-beta-3.toml

```diff
@@ -1,23 +1,23 @@
-date = "2023-07-18"
+date = "2023-08-07"
 
 [pkg.forc]
-version = "0.37.3"
+version = "0.42.1"
```

Comment:
We should only be changing the indexer versions and use old beta-3 versions for all other components.